packages/ui/cart/drawer/CartDrawer.tsx

```diff
@@ -184,6 +190,10 @@ export function CartDrawer() {
                      </Collapse>
                   </DrawerBody>
 
+                  <CheckoutError
+                     error={checkout.error}
+                     onDismiss={checkout.reset}
+                  />
```

Comment:
This doesn't seem terribly related to the rest of the content of this pull. Any particular reason it's in here?

Reply:
Think it was copy-pasted there just for testing out the cart icons, and was not cleaned out